src/components/site-preview.tsx

```diff
@@ -17,25 +20,44 @@ export function SitePreview({ site }: PropsWithChildren<IProps>): JSX.Element {
   const start = useCallback(() => site?.start(), [site])
 
   return (
-    <section>
-      <p>{site?.packageJson?.name ?? `Unnamed site`}</p>
-      <p>Status: {status}</p>
-      {/* TODO: We can do this better by properly keeping track of running status */}
-      {!status || [`STOPPED`, `FAILED`, `INTERRUPTED`].includes(status) ? (
-        <button onClick={start}>Start</button>
-      ) : (
-        <button onClick={stop}>Stop</button>
-      )}
-
+    <Flex
+      as={`section`}
+      // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+      sx={{
+        border: `grey`,
+        borderRadius: 2,
+        flexDirection: `column`,
+        my: 4,
+        p: 4,
+      }}
+    >
+      <Flex css={{ justifyContent: `space-between` }}>
```

Comment:
This can be `justifyContent="space-between"` if we're using strict UI,

Reply:
We're not. Max recommended that I not use it.

Reply:
When I saw it above I couldn't tell which was the intended.